tests/pkg_openwsn/README.md

```diff
@@ -0,0 +1,306 @@
+# OpenWSN on RIOT
+
+This test demonstrates the [OpenWSN](https://github.com/openwsn-berkeley/openwsn-fw) full
+stack (UDP, IPv6, RPL, 6TiSCH) running on RIOT. When flashed, it will initialize the stack
+and provide the user with some minimal shell commands to:
+
+- print the own IPv6 address `ifconfig`
+- change the UDP destination port `udp server start <port>`
+- send a UDP packet `udp send <addr> <port> <data>`
+
+A tun interface can also be set to allow traffic out of the network. This
+allows pinging nodes in the network from the host.
+
+Please note that this port is still in experimental status. For further information
+about the port refer to the [pkg documentation](../../pkg/openwsn/doc.txt).
+
+## Experimental setups
+
+The following setup act as a starting point for testing and debugging. Either
+build and flash local nodes or incorporate the [FIT IoT-LAB](https://www.iot-lab.info/)
+Testbed. Please check the ports [documentation](../../pkg/openwsn/doc.txt) for information
+about supported hardware platforms.
+
+This port currently needs `openvisualizer`, please make sure you follow the
+[pre-requisites](dist/tools/openvisualizer/makefile.openvisualizer.inc.mk)
```

Comment:
I think the link to `makefile.openvisualizer.inc.mk` will be broken with browsing this README on Github.

```suggestion
[pre-requisites](../../dist/tools/openvisualizer/makefile.openvisualizer.inc.mk)
```